controllers/storagecluster/reconcile.go

```diff
@@ -635,47 +635,71 @@ func remove(slice []string, s string) (result []string) {
 	return
 }
 
-// ensureCreated ensures if the osd removal job template exists
-func (obj *ocsJobTemplates) ensureCreated(r *StorageClusterReconciler, sc *ocsv1.StorageCluster) error {
-	osdCleanUpTemplate := &openshiftv1.Template{
+func osdCleanUpTemplate() *openshiftv1.Template {
+
+	NameSpace := "openshift-storage"
```

Comment:
Private/local variables should never be capitalized at the beginning. And in Kubernetes, "Namespace" is one word, not two.

Reply:
done